src/Repository/PaymentMethodRepository.php

```diff
@@ -148,4 +148,16 @@ public function getMethodsForCheckout()
 
         return Db::getInstance()->executeS($sql);
     }
+
+    public function updateTransactionId($oldTransactionId, $newTransactionId)
```

Comment:
why none of methods have return types nor typehints?
What is this module php version support? php 5? :sweat:

Reply:
Yes, module supports 5.6 PHP version